addons/spike/initKeybinds.inc.sqf

```diff
@@ -0,0 +1,80 @@
+["ACE3 Weapons", QGVAR(Designate), LLSTRING(Designate),
+{
+    [SPIKE_KEY_DESIGNATE, true] call FUNC(keyDown);
+},
+{
+    [SPIKE_KEY_DESIGNATE, false] call FUNC(keyDown);
+},
+[15, [false, false, false]], false] call CBA_fnc_addKeybind;  // Tab
+
+["ACE3 Weapons", QGVAR(LeaveCamera), LLSTRING(LeaveCamera),
+{
+    [SPIKE_KEY_LEAVE, true] call FUNC(keyDown);
+},
+{
+    [SPIKE_KEY_LEAVE, false] call FUNC(keyDown);
+},
+[15, [false, true, false]], false] call CBA_fnc_addKeybind;  // Ctrl+Tab
+
+["ACE3 Weapons", QGVAR(PanUp), LLSTRING(PanUp),
+{
+    [CAMERA_KEY_UP, true] call FUNC(keyDown);
+},
+{
+    [CAMERA_KEY_UP, false] call FUNC(keyDown);
+},
+[200, [false, false, false]], false] call CBA_fnc_addKeybind;  //Up Arrow
```

Comment:
To me it makes sense for this and MCLOS to have the same keys, but I'm not sure

```suggestion
[DIK_NUMPAD8, [false, false, false]], false] call CBA_fnc_addKeybind;  // Numpad 8
```

Reply:
I will try to update this to use the new "unified" keybind system
Possibly also look to combine the "lock key" (tab)